language-support/hs/bindings/test/DA/Ledger/Tests.hs

```diff
@@ -668,7 +667,7 @@ runWithSandbox Sandbox{port} maybeAuth tid ls = runLedgerService ls' timeout (co
             Nothing -> ls
             Just authSpec -> setToken (makeSignedJwt authSpec tid) ls
 
-makeSignedJwt :: AuthSpec -> TestId -> Jwt
+makeSignedJwt :: AuthSpec -> TestId -> String
```

Comment:
Here you are also inconsistent with the type synonym. You get back `String` from this function and the only use site is `setToken` which expects `Tok`.